var/spack/repos/builtin/packages/buildah/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Buildah(MakefilePackage):
+    """Lightweight tool for building Open Container Initiative (OCI) container images"""
+
+    homepage = "https://buildah.io/"
+    git = "https://github.com/containers/buildah"
+
+    maintainers("upsj")
+
+    license("Apache-2.0", checked_by="upsj")
+
+    version("1.37.5", commit="5fd40b989860984a00f6fc1539ff53caceca1325")
```

Comment:
```suggestion
    version("1.37.5", tag="v1.37.5", commit="5fd40b989860984a00f6fc1539ff53caceca1325")
```

Suggestion to add the `v1.37.5` tag. This allows Spack to more quickly clone the ref from GitHub.